feat/CLET-[282/284] add 3ds related info to cards section (#247)
FidelLimited/docs: stream/cards.md

### What Is Card Verification

When linking a card, cardholders are required to complete one of the two available verification processes to verify card ownership to ensure we protect them.

> When linking a card, a property `verificationMethod` will identify what verification method that card is meant to follow - `3ds` or `microcharge`.

Reviewer: I am not sure if adding this information will be helpful or if it will cause more confusion. Since it's an internal property, I don't think it's valuable to be stated here. We could simply say, 'When linking a card, two verification methods may be available: 3DS or Microcharge.' Wdyt?

> When linking a card, a property `verificationMethod` will identify what verification method that card is meant to follow - `3ds` or `microcharge`.

Reviewer: does it make sense to explain that this property is returned from the create card endpoint?

Author: yeah, I'll add a note

### 3DS (3D Secure)

3D Secure is a card verification method that is meant to enhance security measures for shoppers and vendors alike. This verification process consists of the following steps:

1. The cardholder links their card to a Fidel API program using the Verified Enrollment SDK or via the Create Card and the Create Consent endpoints.

> 2. A new screen prompting for additional information will be shown (i.e Cardholder's name and CVV/CVC)

Reviewer 2: I think “a new screen” implies using our web SDK or it can imply our customer needing to present a new screen (which is not necessarily true, if the customer already knows that a Mastercard is being enrolled). I think this page should have the Fidel API as the first class citizen and refer to a Fidel API integration. So, an alternative formulation for this point would be: when calling the Create Consent endpoint, you would need to provide more cardholder details. Something like that.

Author: Yeah, that's a fair point, I'll try to change it accordingly

3. After inputting that information:
    1. If the verification is required, it should proceed to the next steps
    2. If no extra verification is required, the card consent will be verified
4. Fingerprinting process will take place (this is part of the network's requirements)
5. When the fingerprinting process completes:
    1. If there is no need for further verification (frictionless), card consent will be verified
    2. If there is a need for challenge (consent `status` is `awaiting` ) a challenge prompt will be shown, for the cardholder to input a code from a known device (2FA)
6. After inputting the correct code, consent will be verified

> If for some reason you cancel the process while in the verification process, you will have to wait an hour for the consent to be automatically revoked (for security reasons).

Reviewer 3: Shouldn't "you" here be "the cardholder"?

> If for some reason you cancel the process while in the verification process, the cardholder will have to wait an hour for the consent to be automatically revoked (for security reasons).

Reviewer 3: ```suggestion
If for some reason the verification process is canceled, the cardholder will have to wait an hour for the consent to be automatically revoked (for security reasons).
```

Reviewer 3: Unsure if "canceled" or "interrupted" makes more sense

Author: interrupted might make more sense, yeah..

- The cardholder has a maximum of three attempts to confirm the charged amount, after which they will be blocked from further attempts.
- Cards that are not verified will be purged from the Fidel API system 48 hours after linking.
- If the cardholder has any issues with card linking and verification, the card can be deleted from the Dashboard to restart the linking from the beginning, or can be automatically purged after 48 hours.

> - When linking a Fidel API test card, type in $0.67 for verification (if `verificationMethod` is `microcharge`).

Reviewer 3: Do we need the "if" when this section is already under Microcharge?

Author: good point, probably not... I'll remove it

When linking a Fidel API test card, type in `$0.67` for verification.

For 3D Secure, you may use one of these test card numbers:

> 1. No verification required - `5186002600000005`

Reviewer 2: Are these test cards that can be used with `pk_test` keys?

Reviewer: Only with `live` keys, I think 🤔

Author: `pk_live`, I'm afraid, is due to a "test" logic they have on the endpoint, it needs to be live

Reviewer 2: Are we allowed to make them public? I would think that we are not allowed as these are test cards coming from MC made available only for our use.

    "expMonth": 10,
    "expYear": 2025,
    "countryCode": "USA"
    "cvc": "123", // required if verificationMethod is 3ds

>     "cardholderName": "John Doe" // required if verificationMethod is 3ds

Reviewer 3: Should we rather have two "create consent" sections? One for each method?

Reviewer 3: Also, should we have a "Fingerprint" section as well? That's a new endpoint

Author: Do you think it'll make more sense? I'd say that it'll only make the page bigger and this would be clear on the API reference, I think 🤔

-H 'Content-Type: application/json' \
-H 'Fidel-Key: pk_test_62f02030-0409-4eb5-ab94-6eff05b3d888' \
-d '{

>     "code": 0.67 // only if verificationMethod is 3ds

Reviewer 3: Microcharge
